test/gibct/reducers/autocomplete.unit.spec.js

```diff
@@ -0,0 +1,100 @@
+import { expect } from 'chai';
+
+import autocompleteReducer from '../../../src/js/gi/reducers/autocomplete.js';
+
+const initialState = {
+  inProgress: false,
+  previewVersion: null,
+  searchTerm: '',
+  facilityCode: null,
+  suggestions: []
+};
+
+describe('autocomplete reducer', () => {
+  it('should change autocomplete term', () => {
+    const state = autocompleteReducer(
+      { ...initialState, facilityCode: 'code' },
+      {
+        type: 'AUTOCOMPLETE_TERM_CHANGED',
+        searchTerm: 'newSearchTerm',
+      }
+    );
+
+    expect(state.facilityCode).to.be.eql(null);
```

Comment:
Nitpick: I actually meant it would read better as `to.eql` than `to.be.eql`. As in, "this value should equal [value]".